### ISSUE TITLE
aws_rds_global_cluster does not support aurora-postgresql

### ISSUE THREAD

**Issue by singh1469** [terraform-providers/terraform-provider-aws#12382]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
```
Terraform v0.12.23
+ provider.aws v2.52.0
+ provider.local v1.4.0
+ provider.null v2.1.2
+ provider.random v2.2.1
+ provider.template v2.1.2
+ provider.tls v2.1.1
```

### Affected Resource(s)

* aws_aurora_global (postgreSQL)

### Terraform Configuration Files

```hcl
resource "aws_rds_global_cluster" "kong" {
  engine = "aurora-postgresql" # error-producing line
  global_cluster_identifier = "my_global"
}
```

### Debug Output

https://gist.github.com/singh1469/9c0ebaa2487840cb10542c64733ea861

### Expected Behavior

Aurora global database now supports PostgreSQL as documented here:
https://aws.amazon.com/about-aws/whats-new/2020/03/amazon-aurora-with-postgresql-compatibility-supports-amazon-aurora-global-database/
So I should be able to create a global Aurora cluster with the engine set to: aurora-postgresql

### Actual Behavior

Terraform plan returns an error that prevents the creation of a global aurora cluster with engine = aurora-postgresql

I'm able create this setup fine using the AWS CLI without any problems

### Steps to Reproduce

1. Create a main.tf file and add the following contents:
```
resource "aws_rds_global_cluster" "kong" {
  engine = "aurora-postgresql"
  global_cluster_identifier = "kong"
}
```
2 Run `terraform plan` or `terraform apply` to produce the error. Please note on older versions of Terraform, this error will only appear when running `terraform apply`

**Comment by bflad** [comment]
Pull request submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/12401

**Comment by singh1469** [comment]
Thanks guys!

**Comment by bflad** [comment]
Support for this has been merged and will release with version 2.57.0 of the Terraform AWS Provider, later today. 👍
